transactions.asciidoc

```diff
@@ -360,6 +356,74 @@ While the zero address is only intended for contract registration, it sometimes
 Any ether sent to the contract registration address +0x0+ or the designated burn address +0x0...dEaD+ above will become unspendable and lost forever.
 ====
 
+A contract registration transaction should contain no ether value, only a data payload that contains the compiled bytecode of the contract. The only effect of this transaction is to register the contract.
```

Comment:
Perhaps for this we need to revisit "Sending value in a contract creation" #313 wisely opened by @nicksavers 
I mean, something like **"should not, but could"**.

Reply:
I have not written this line, so I did not want to change it. I have only moved its order. After the original `TODO` there was a paragraph about a specific address for burning ether. I have moved it closer to the explanation about address `0x0` since I thought it will be strange to  return to it after a long example. 

Regardless, I was not able to create the `Faucet.sol` contract with a transaction where `value != 0`

Reply:
I see, I got it, sorry and Thanks @amanusk